Makefile

```diff
@@ -1,19 +1,52 @@
+# Copyright 2020 Joyent, Inc.
+
 UNAME=$(shell uname -s | tr "[:upper:]" "[:lower:]")
 VER=$(shell json -f package.json version)
+NAME := thoth
+RELEASE_TARBALL := $(NAME)-$(UNAME)-$(VER).tar.gz
+ROOT := $(shell pwd)
 
-node_modules: package.json
-	npm install --production
-	@touch node_modules
+NODE_PREBUILT_VERSION=v8.17.0
+NODE_PREBUILT_TAG=gz
+NODE_PREBUILT_IMAGE=5417ab20-3156-11ea-8b19-2b66f5e7a439
 
-.PHONY: release github-release
-release: thoth-$(UNAME)-$(VER).tar.gz
+ENGBLD_USE_BUILDIMAGE = false
+ENGBLD_SKIP_VALIDATE_BUILDENV = true
+ENGBLD_REQUIRE := $(shell git submodule update --init deps/eng)
+include ./deps/eng/tools/mk/Makefile.defs
+TOP ?= $(error Unable to access eng.git submodule Makefiles.)
+RELSTAGEDIR := /tmp/$(NAME)-$(STAMP)
 
-publish: release
-	./tools/publish.sh "$(VER)"
+include ./deps/eng/tools/mk/Makefile.node_prebuilt.defs
+
+.PHONY: all
+all: $(NPM_EXEC)
+	$(NPM) install
 
-thoth-$(UNAME)-$(VER).tar.gz:
-	./tools/build-release.sh
+DISTCLEAN_FILES += ./node_modules
+
+.PHONY: release
+release: all
+	@echo "Building $(RELEASE_TARBALL)"
+	@mkdir -p $(RELSTAGEDIR)/site
+	@touch $(RELSTAGEDIR)/site/.do-not-delete-me
+	@mkdir -p $(RELSTAGEDIR)/opt/custom/thoth
+	cp -r \
+	    $(ROOT)/analyzers \
+	    $(ROOT)/build \
```

Comment:
If you want to limit the tarball size, there is a fair amount under build/... that need not be included. For example, you'll be getting the sdcnode tarball *and* the cracked tarball. You'll be getting the sdcnode's `npm` and its libs that aren't needed for runtime.

Reply:
Ah, thanks!